### PR TITLE
Rename assets-hugo directory to assets

### PR DESCRIPTION
**Pull request by anoadragon453**
Now that we've dropped the old build pipeline (and an `assets` directory does not exist in the repo any longer, we can rename the hugo version of the assets (`assets-hugo`) created during the build tools migration back to simply `assets`.

### CODE REVIEW

File: config.toml [hunk 1]
```diff
@@ -12,10 +12,6 @@ theme = ["docsy"]
 
 disableKinds = ["taxonomy", "taxonomyTerm"]
 
-# Change the default for assets, because the old Python toolchain uses "assets" for build output.
-# When the old toolchain is retired we can switch back to the default here.
-assetDir = "assets-hugo"
```
Comment (anoadragon453) [inline config.toml:17]: The default for `assetDir` is "assets": https://gohugo.io/getting-started/configuration/#all-configuration-settings

**Review by richvdh — CHANGES_REQUESTED** [review]
looks fine, but we should probably remove `scripts/generate-matrix-org-assets` at the same time?

**Comment by richvdh** [comment]
@anoadragon453 is this still on your radar?

**Comment by anoadragon453** [comment]
Yes, just a tuit.

**Comment by afranke** [comment]
> looks fine, but we should probably remove `scripts/generate-matrix-org-assets` at the same time?

Just saw this. Already removed as part of b55cc15772c162dd7bcc1c6d59035d89601fb1f8

**Comment by richvdh** [comment]
@afranke shall we just merge this then?

**Comment by afranke** [comment]
Actually no, we shan’t. The [build-openapi step makes an archive of the `assets` directory](https://github.com/matrix-org/matrix-doc/blob/dfec7d687ee9e0ae507b9eee4b44af70ffd56d2c/.github/workflows/main.yml#L80) and uploads that as artifact, expecting it to contain only the generated definition. One would need to tweak the workflow first so it uses another directory name for that job.

Considering the branch here does not include these changes to the workflow, and Github does not like rebases, I’ll create another PR to do that and it should be reviewed and merged before this one gets merged.

**Review by afranke — APPROVED** [review]
With #3621 merged, I reckon this is good to go now.

**Pull request merged**